### PR TITLE
Method removeApiToken

### LINKED ISSUE #2: Method getApiTokens return type is wrong
We have to update the return type of the method getApiTokens

https://github.com/poetapp/frost-client/blob/master/src/Frost.ts#L291

Now the typedef return is the following

```
  getApiTokens(token: string): Promise<ReadonlyArray<{
        apiToken: string;
        dateCreated: string;
    }>>

```
but isn't correct, now the API return only the tokens. Therefore, we have to update the typedef return for:

```
 getApiTokens(token: string): Promise<{ apiTokens: ReadonlyArray<string> }>
```

Also, we have to update the Frost web where use this method

https://github.com/poetapp/frost-web/blob/master/src/sagas/GetApiTokens.ts#L8

### PR DESCRIPTION
**Pull request by wzalazar**
This PR introduces

- new version 0.1.4
- adds method removeApiToken
- fix return type of getApiTokens
- fixes HTTP method for API fetch, ```delete``` instead of ```del```

fix #2

### CODE REVIEW

File: src/Frost.ts [hunk 1]
```diff
@@ -312,6 +310,28 @@ export class Frost {
...
+      method: Method.DEL,
+      headers: new Headers({
+        'Content-Type': 'application/json',
+        token
+      })
+    }
+
+    const request = fetch(`${this.host}${Path.TOKENS}/${tokenId}`, options)
+
+    return Promise.race([request, this.timeoutPromise()])
```
Comment (lautarodragan) [inline src/Frost.ts]: I'm wondering if we could avoid the use of `.then` by `await`ing the result of `Promise.race`...? That way we'd keep the code consistent everywhere.

```
const response = await Promise.race....
// if the timeout "won" the race, it'd have thrown, so we wouldn't reach the following line of code.
if (response.ok) return await value.text()
throw await value.text()
```

Something like this?

Comment (wzalazar) [reply src/Frost.ts]: yes, it's possible, I will apply a refactor if you want...

Comment (lautarodragan) [reply src/Frost.ts]: Yes please

Comment (lautarodragan) [reply src/Frost.ts]: (But only on this case, if there's a big refactor involved to things outside the scope of this PR, let's leave it to a future PR)

**Pull request merged**